### PR TITLE
update-topics

### PR DESCRIPTION
**Pull request by Wladefant**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> More up to date and neater

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/arduino/index.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-created_by: Massimo Banzi
+created_by: Massimo Banzi, David Cuartielles, Tom Igoe, Gianluca Martino
 display_name: Arduino
-github_url: https://github.com/arduino/
+github_url: https://github.com/arduino
 logo: arduino.png
-released: '2003'
-short_description: Arduino is an open source hardware and software company and maker
-  community.
+related: electronics, microcontroller, programming, raspberry-pi
```
Comment (smashwilson) [inline topics/arduino/index.md:6]: "Programming" is too vague to be here.

```suggestion
related: electronics, microcontroller, raspberry-pi
```

File: topics/motorola-68000/index.md [hunk 2]
```diff
@@ -13,4 +13,4 @@ The Motorola 68000 is a 16/32-bit microprocessor (family) designed by Motorola i
 
 The 68000 was used in many personal computers, workstations, and game consoles, including the Apple Macintosh, Atari ST, Commodore Amiga, and the Nintendo Entertainment System.
 
-The 68000 was succeeded by the 68020, 68030, 68040, and 68060, and by the PowerPC 601, 603, 604, 620, and 750.
+The 68000 was succeeded by the 68020, 68030, 68040, and 68060, and by the PowerPC 601, 603, 604, 620, and 750.
```
Comment (smashwilson) [inline topics/motorola-68000/index.md:16]: Hah! Thanks for cleaning this up 😁

**Review by smashwilson — COMMENTED** [review]
Overall these changes look pretty good! I'm a fan of the expanded and clarified descriptions.

We do try not to overuse the "related" topics feature... otherwise, virtually every topic could be related to "software", for example, which wouldn't be very useful. We also try to make sure that the topics we list as aliases or related are already being used in the wild (we don't add "speculative" topic associations). In fact, if you find places where we have let empty topics slip in, you could do us a favor by taking them out 😎 

One more thing - in general, we prefer to split things up so that each PR touches a single topic (or collection) when possible. It makes it easier to review and your changes get in more quickly, because we could merge the ones for the topic changes you've proposed that we _don't_ have suggested changes for while we're going back and forth on the ones we do. There are exceptions to that but I think it would really help these kinds of maintenance updates!

Thanks so much for all of the effort you're putting in here! I'll be happy to merge this in once we resolve all the suggestion threads.

**Comment by Wladefant** [comment]
Understood ;). Thank you for the help!!! Resolved all conflicts

File: topics/3d/index.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-short_description: 3D modeling is the process of virtually developing the surface
-  and structure of a 3D object.
+short_description: 3D refers to the use of three-dimensional graphics, modeling, and animation in various industries.
+related: graphics, animation, modeling, cad, printing, computer-graphics, product-design, video-games
 topic: 3d
-wikipedia_url: https://en.wikipedia.org/wiki/3D_modeling
+wikipedia_url: https://en.wikipedia.org/wiki/3D_computer_graphics
 ---
-3D modeling uses specialized software to create a digital model of a physical object. It is an aspect of 3D computer graphics, used for video games, 3D printing, and VR, among other applications.
+3D technology is used in a wide range of fields, including film, video games, architecture, engineering, and product design. It allows designers to create digital models of object s that can be manipulated and rendered in three dimensions. 3D modeling software is used to create and manipulate 3D models, and 3D animation software is used to create movement and effects within those models. 3D technology has also been adopted for use in 3D printing, where physical objects can be created from digital models.
```
[the remark this answers is not in the document]
Comment (smashwilson) [reply topics/3d/index.md:9]: 🤔 These are marked as resolved, but the diffs don't actually appear to be applied. Did you mean to apply the suggestions instead?

**Review by ABoutPapper — COMMENTED** [review]
The Modeling, graphics of the Animation's for all various Industries look great.

**Review by gumafia — COMMENTED** [review]
ทำตามอำเภอใจไปหรือปล่าว แล้วผมจะแก้ไขอะไรได้บ้างเหรอ แต่ก็ดีไม่ต้องแก้อะไรเพราะกูไม่รู้จะทำยังไง แอนดรอย2สายพันธ์

**Comment by gumafia** [comment]
ทำอย่างนี้จะอยู่ในเครื่องเพื่ออะไรพอแสดงความเห็นหน่อยก็ล้มเลิกที่ปรับปรุงเลยเหรอ

File: topics/3d/index.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Wladefant) [reply topics/3d/index.md:9]: Yeah sorry

**Review by kenyonj — APPROVED** [review]
Looks great. Thanks for your contribution!

**Pull request merged**